```c
void perror( char const *message );
```
- 如果message不是NULL并且指向一个非空的字符串，`perror`函数就打印出这个字符串，后面跟一个分号和一个空格，然后打印出一条用于解释`errno`当前错误代码的信息
- 只有当一个库函数失败时，才会设置`errno`。当函数成功运行时，`errno`的值不会被修改。所以我们不能通过测试`errno`的值来判断是否有错误发生，只有当被调用的函数提示有错误发生时，检查`errno`的值才有意义